tools/replace_code_block.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+from traceback import print_tb
+from typing import AnyStr
+
+
+def check_for_code(filename: AnyStr, check: bool, print_not_fixed: bool):
```

Comment:
Can we drop the `print_not_fixed` part? I don't think it adds much value.

Reply:
I wanted to make it similar to black, but maybe it should be renamed to `verbose_mode`

Reply:
Let's just drop it. Keep it simple so we can get this in.

Reply:
Dropped.